src/main/java/org/example/service/DeveloperService.java

```diff
@@ -0,0 +1,55 @@
+package org.example.service;
+
+import org.example.model.Developer;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DeveloperService {
+
+    private static final String GET_DEVELOPERS = "select user_id, firstname, lastname, age, proglang from developer";
+
+    private final ConnectionService connectionService = ConnectionService.getInstance();
+
+    private static DeveloperService INSTANCE = null;
```

Comment:
Remove null.

Check the Java conventions for naming:

- variables
- methods
- classes
- constant variables
- class files

Reply:
fix